qualtran/bloqs/util_bloqs_test.py

```diff
@@ -107,30 +84,39 @@ def build_composite_bloq(self, bb: 'BloqBuilder', test_regs: 'SoquetT') -> Dict[
 
 
 def test_partition():
-    bloq = TestPartition(test_bloq=TestCNOT())
+    bloq = TestPartition(test_bloq=CNOT())
     assert_valid_bloq_decomposition(bloq)
-    bloq = TestPartition(test_bloq=ComplicatedBloq())
+
+    bloq = TestPartition(test_bloq=TestMultiRegister())
     assert_valid_bloq_decomposition(bloq)
 
 
 def test_partition_tensor_contract():
-    bloq = TestPartition(test_bloq=ComplicatedBloq())
+    bloq = TestPartition(test_bloq=TestMultiRegister())
     tn, _ = cbloq_to_quimb(bloq.decompose_bloq())
     assert len(tn.tensors) == 3
-    assert bloq_to_dense(bloq).shape == (2048, 2048)
+    assert bloq_to_dense(bloq).shape == (4096, 4096)
 
 
 def test_partition_as_cirq_op():
-    bloq = TestPartition(test_bloq=TestCNOT())
+    bloq = TestPartition(test_bloq=CNOT())
     quregs = get_named_qubits(bloq.signature.lefts())
     op, quregs = bloq.as_cirq_op(cirq.ops.SimpleQubitManager(), **quregs)
     unitary = cirq.unitary(cirq.Circuit(op))
     assert np.allclose(unitary, bloq_to_dense(CNOT()))
-    bloq = TestPartition(test_bloq=ComplicatedBloq())
-    quregs = get_named_qubits(bloq.signature.lefts())
-    op, quregs = bloq.as_cirq_op(cirq.ops.SimpleQubitManager(), **quregs)
-    unitary = cirq.unitary(cirq.Circuit(op))
-    assert np.allclose(unitary, bloq_to_dense(bloq))
```

Comment:
Where did the unitary test go?

Reply:
the factored out `TestMultiRegister` uses the testing bloqs which don't have cirq equivalents. The first half of this test does a unitary comparison; I added a diagram check, which should validate the `Partition` behavior